tests/accordion-test.ts

```diff
@@ -0,0 +1,605 @@
+                    body: "body 2",
+                    active: false,
+                },
+                { title: "title 3?", body: "body 3", active: true },
+                { title: "title 4?", body: "body 4", active: true },
+                { title: "title 5?", body: "body 5", active: false },
+            ])
+        );
+
+        const titles = el.querySelectorAll(".acc-title");
+        const containerElement = el.querySelectorAll(".container");
+        const bodysel = el.querySelectorAll(".acc-body")
+        expect(el.hasAttribute("nomultiple")).to.be.false;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(3);
+        const active = el.querySelectorAll(".acc-active");
+        containerElement.forEach((_,index) =>{
+            if(index === 0 || index === 2 || index === 3){
+                expect(getComputedStyle(bodysel[index]).display).to.equal("block")
+            }
+            else {
+                expect(getComputedStyle(bodysel[index]).display).to.equal("none");
+            }
+        })
+
+        titles[0].click();
+        await el.updateComplete;
+        titles[1].click();
+        await el.updateComplete;
+        titles[2].click();
+
+        await el.updateComplete;
+        expect(titles[0].parentElement?.classList.contains("acc-active")).to.be
+            .false;
+        expect(titles[1].parentElement?.classList.contains("acc-active")).to.be
+            .true;
+        expect(titles[2].parentElement?.classList.contains("acc-active")).to.be
+            .false;
+        expect(titles[3].parentElement?.classList.contains("acc-active")).to.be
+            .true;
+
+        containerElement.forEach((_,index)=>{
+            if(index === 1 || index === 3) {
+                expect(getComputedStyle(bodysel[index]).display).to.equal("block");
+            }
+        })
+        
+        expect(el.querySelectorAll(".acc-active").length).equal(2);
+        titles[1].click();
+        await el.updateComplete;
+        titles[3].click();
+        await el.updateComplete;
+        el.setAttribute("nomultiple", "");
+        await el.updateComplete;
+
+        titles[2].click();
+        await el.updateComplete;
+        titles[1].click();
+        await el.updateComplete;
+
+        expect(el.querySelectorAll(".acc-active").length).equal(1);
+        expect(el.hasAttribute("nomultiple")).to.be.true;
+        const activeItemsAfter = el.querySelectorAll(".acc-active");
+
+        console.log(el);
+        activeItemsAfter.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        containerElement.forEach((_,index)=>{
+            if(index !== 4){
+                expect(getComputedStyle(bodysel[index]).display).to.equal("block")
+            }
+            else {
+                expect(getComputedStyle(bodysel[index]).display).to.equal("none");       
+            }
+        })
```

Comment:
This seems to check for three elements to have display:block and one element with display:none. But nomulitple is set to true just above this validation. Seems to be wrong.